packages/flutter_test/lib/src/window.dart

```diff
@@ -1411,30 +1724,55 @@ class TestWindow implements SingletonFlutterWindow {
   /// Deletes any existing test accessibility features and returns to using the
   /// real accessibility features.
   @Deprecated(
-    'Use platformDispatcher.clearAccessibilityFeaturesTestValue() instead. '
+    'Use WidgetTester.platformDispatcher.clearAccessibilityFeaturesTestValue() instead. '
     'This feature was deprecated after v2.11.0-0.0.pre.'
   )
   void clearAccessibilityFeaturesTestValue() {
     platformDispatcher.clearAccessibilityFeaturesTestValue();
   }
 
+  @Deprecated(
+    'Use WidgetTester.platformDispatcher.onAccessibilityFeaturesChanged instead. '
+    'Deprecated to prepare for the upcoming multi-window support. '
+    'This feature was deprecated after v3.9.0-0.1.pre.'
+  )
   @override
   VoidCallback? get onAccessibilityFeaturesChanged => platformDispatcher.onAccessibilityFeaturesChanged;
+  @Deprecated(
+    'Use WidgetTester.platformDispatcher.onAccessibilityFeaturesChanged instead. '
+    'Deprecated to prepare for the upcoming multi-window support. '
+    'This feature was deprecated after v3.9.0-0.1.pre.'
+  )
   @override
   set onAccessibilityFeaturesChanged(VoidCallback? callback) {
     platformDispatcher.onAccessibilityFeaturesChanged = callback;
   }
 
+  @Deprecated(
+    'Use WidgetTester.view.updateSemantics instead. '
```

Comment:
This is a method that accepts a parameter. Should the deprecation message reflect that? Perhaps:

```suggestion
    'Use WidgetTester.view.updateSemantics(update) instead. '
```

(I don't know what the best practice is, please double-check my suggestion as it may be wrong)

Reply:
Same comment for similar methods below including `setIsolateDebugName`, `sendPlatformMessage`, etc...

Reply:
The parameters were actually why I _removed_ the `()`s originally. I didn't know how to represent them in the message, so I just made it consistently without `()`s everywhere...

@goderbauer, do you know of any official guidance on how to represent parameters in these messages? Otherwise I'll go with @loic-sharma's suggestion here.

Reply:
As far as I know, there's no official guidance. I like Loïc's suggestion. Just `WidgetTester.view.updateSemantics()` would work as well just to indicate that it is a method...

Reply:
Done!